src/cli/utils/suggestions.go

```diff
@@ -0,0 +1,115 @@
+package utils
+
+// The code in this file is mostly lifted out of cobra itself. It as
+// of now has  some issue with how it handles completions for
+// subcommands and only autocompletes for top level commands.
+// https://github.com/spf13/cobra/issues/981#issuecomment-547003669
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// SubcommandsRequiredWithSuggestions will ensure we have a subcommand provided by the user and augments it with
+// suggestion for commands, alias and help on root command.
```

Comment:
Could you please add some more inline comments on what this function is doing, especially around subcmd alias suggestions? :)